### PR TITLE
Finished 10 nouns + action exercises.

### PR DESCRIPTION
**Pull request by michaelmon1203**
Slack username:
michaelmon1203

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun:        string
+//  action 1:    string vibrates
+string.vibrate();
+//  action 2:    string rings
+string.ring();
+//  action 3:    string breaks
+string.break();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (michaelmon1203) [reply main.cpp]: Sorry I didn't get it. I think I followed the requirement strictly.

like for
1) write out 10 nouns in the space provided below.
    Choose nouns that do things.  
    avoid choosing nouns that have things done TO them.
    If your noun consists of multiple words, use camelCaseToNameIt.

I did it, and did used multiple words noun in my second exercise, like cutoffFreq, volumeKnob. 


and for
 come up with a new noun and 3 actions
 pick a noun that needs camelCase for the name
 pick actions that need camelCase for their name

You didn't say I must make all noun consist of multiple words. And where did you mention I have to pick all actions with camelCase name? I re-checked the gumroad video and the description you left on the project, I only find these requirements:

 2) for each of the 10 nouns, write out 3 actions it might perform, in plain English.
 3) write out how you'd call that action in pseudo code, in the space after the plain-english action
 4) If the action requires multiple words, use camelCaseToNameIt

It's an IF, not a requirement for every action name.
So I did it in couple of my exercises. 


And for
Do not create an 11th noun.

I strictly followed the guide and wrote out 10 exercises in the blank part you left with numbers like 1) 2), you can count it out there're 10 added. The 11th one you might refer to the one YOU left in the example, like arm.extend() , you left it, it's not me. Maybe you want me to delete it? Then why don't you make it clearer if it's required?

Sorry my English is not that good, but I have tried my best to follow every step and requirement you asked, hope I explained it well, I couldn't find out what should I fix for this merge.

Comment (michaelmon1203) [reply main.cpp]: Oh, I think I might get it, like it's not my coding mistake, it's a process to learn pull request?

**Review by matkatmusic — APPROVED** [review]
